### ISSUE TITLE
social warfare and bitly api v4

### ISSUE THREAD

**Issue by emyabdel** [warfare-plugins/social-warfare#754]
Hi, I received an email from bitly to move to API V4. I use social warfare Bitly Link Shortening
that uses always the old API. What to do to migrate to API V4 as they ask.

**Comment by Nicholas-Cardot** [comment]
The Bitly API v4 has already been integrated into the plugin, but this version of the plugin has not yet been released. However, if you use Bitly and want to get the new version of the API on your site, you can simply contact our customer support and they will send you the latest copy of the plugin. 

The new Bitly setup in that version of the plugin also contains a date field that you can set to avoid creating large numbers of shortened URL's. So you set the date in that field, and any posts older than that date will not get shortened links. If you want all posts to have links, then you set that date to something really, really old.

**Bonus:** This version of the plugin also contains lots of cool new features for you to check out too.

**In summary:** Simply contact support and they'll send you a copy of Social Warfare 4.0. This version has all of the updated Bitly API stuff that you'll need.

**Comment by emyabdel** [comment]
ok thank you
